### PR TITLE
added shuffled_array.py file to create a shuffled list from given list

### PR DESCRIPTION
**Pull request by manu-prakash-choudhary**
### Describe your change:

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by manu-prakash-choudhary** [comment]
I have created an algorithm to shuffle an array where the seed value to shuffle is provided using pseudo random number generator.

File: other/shuffled_array.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+To create a shuffled array. Generate a pseudo random number(pseudo number is generate with random time provided as its seed value) , Swap that number with any other element in the list run this swapping len(Array) times to get desired shuffled array
+
+To Understand more of random number generation follow https://en.wikipedia.org/wiki/Lagged_Fibonacci_generator
+"""
+
+import random
+import time
+class Solution:
+    def __init__(self,array:list)->None :
+        """
```
Comment (CaedenPH) [inline other/shuffled_array.py:11]: Docstrings should never go in the `__init__`, but in the class itself

File: other/shuffled_array.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
+"""Hello World 
+To create a shuffled array. Generate a pseudo random number(pseudo number is generate with random time provided as its seed value) , Swap that number with any other element in the list run this swapping len(Array) times to get desired shuffled array
+
+To Understand more of random number generation follow https://en.wikipedia.org/wiki/Lagged_Fibonacci_generator
+"""
+
+import random
+import time
+class Solution:
```
Comment (CaedenPH) [inline other/shuffled_array.py:9]:
```suggestion

class Solution:
```

File: other/shuffled_array.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        """
+        >>> Solution().__init__(array=[1,2,3])
+        Traceback (most recent call last):
+                   ....
+        TypeError: __init__() missing 1 required positional argument: 'array'
+        """
+        self.arr = array    
+        self.seed = int(str(time.time())[-1:-5:-1])                  #generating a 4 digit number randomly using decimals of current time
+
+    def prng(self,num:int)->int:                           #pseudo random number generator
```
Comment (CaedenPH) [inline other/shuffled_array.py:20]: Document this function better

**Review by CaedenPH — CHANGES_REQUESTED** [review]
Please format this code with black

**Comment by manu-prakash-choudhary** [comment]
I have made the suggest changes in documentation spacing and doctest

**Comment by manu-prakash-choudhary** [comment]
@CaedenPH please approve my request I made the required changes

**Pull request closed without merging**